man/build_class.py

```diff
@@ -76,7 +74,6 @@ def build_class(ext):
         year = sys.argv[3]
 
     from build import (
-        grass_version,
```

Comment:
Where did the usage go away? It doesn't seem from this PR

Reply:
it is, line 43